Description: Fix trivial test cases to work
 Includes cases of:
 - add more "..." wildcard patterns to the test
 - changing the value to an exactly-equivalent value. 
 These should probably be forwarded upstream.
 .
 Also includes cases of:
 - update an expected error message to some conceptually identical message
   that occurs due to build-dep version differences between us and Sage
 These may or may not need to be forwarded upstream.
 .
 DO NOT INCLUDE:
 - differences in algebraic expansions that are conceptually identical
 - any minute concrete non-identical floating point differences
 since these might indicate deeper bugs and we don't want to hide those.
Author: Ximin Luo <infinity0@debian.org>
Forwarded: TODO
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/interfaces/tachyon.py
+++ b/sage/src/sage/interfaces/tachyon.py
@@ -173,7 +173,7 @@
             sage: from sage.interfaces.tachyon import TachyonRT
             sage: t = TachyonRT()
             sage: t.usage(use_pager=False)
-            Tachyon Parallel/Multiprocessor Ray Tracer   Version...
+            Usage: tachyon modelfile [options]...
         """
         r = os.popen('tachyon').read()
         if use_pager:
--- a/sage/src/sage/tests/books/judson-abstract-algebra/sylow-sage.py
+++ b/sage/src/sage/tests/books/judson-abstract-algebra/sylow-sage.py
@@ -226,7 +226,7 @@
 ~~~~~~~~~~~~~~~~~~~~~~ ::
 
     sage: gap.version()
-    '4.10.0'
+    '4...'
 
 ~~~~~~~~~~~~~~~~~~~~~~ ::
 
